homework7.js

```diff
@@ -0,0 +1,125 @@
+// 1) чотири класи для створення об'єктів-сутностей (це можуть бути тварини, покемони, раси і т.д. - проявіть фантазію)
+
+// 2) у кожного класу має бути мінімум 3 властивості та мінімум 3 методи(але можна й більше)
+
+// 3) у кожного класу має бути своя унікальна властивість
+
+// 4) у кожного класу має бути приватна властивість
+
+// 4) у двох класів має бути спільний предок та спільний метод характерний тільки для них
+
+// 5) у всіх чотирьох класів має бути один (крім проміжних) клас-предок
+
+class lotrCharacter {
+	constructor(name, weaponOfChoice, skills) {
+		this.name = name;
+		this.weaponOfChoice = weaponOfChoice;
+		this.skills = skills;
+	}
+}
+
+class Man extends lotrCharacter {
+	#corruptedByTheRing = false;
```

Comment:
```suggestion
	#corruptedByTheRing;
```
Приватні змінні мають ініціюватись в середині конструктора